src/main/java/com/redhat/cloud/notifications/auth/RHIdentityAuthMechanism.java

```diff
@@ -51,7 +49,7 @@
                     good = true;
                 }
             } else if (path.startsWith("/openapi.json") || path.startsWith("/internal")
-                    || path.startsWith("/admin") || path.startsWith("/health")) {
+                    || path.startsWith("/admin") || path.startsWith("/health") || path.startsWith("/q/health")) {
```

Comment:
Same should apply for /metrics and /q/metrics.
Or we do:

```
# Quarkus since 1.11 redirects non-apps to /q/. We need to prevent this
quarkus.http.redirect-to-non-application-root-path=false
quarkus.http.non-application-root-path=/
```

Reply:
I wouldn't change the default Quarkus behavior without a good reason, so I'm in favor of adding `/metrics` and `/q/metrics` to the list.

Reply:
The /q thing can be tricky, as we learned with openapi. If you request /openapi.json, it will return a 302 redirect which clients (sometimes) can't understand

Reply:
That sounds like the good reason I needed 😄

Reply:
See POL-443